packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarTextButtonIOS.ios.js

```diff
@@ -89,7 +91,12 @@ const BpkNavigationBarTextButtonIOS = (props: Props) => {
   const accessibilityStates = [];
   if (disabled) {
     accessibilityStates.push('disabled');
-    tintColorFinal = disabledTintColor || colorSkyGrayTint04;
+    tintColorFinal =
+      disabledTintColor ||
+      useBpkDynamicValue({
```

Comment:
Hooks should not be used inside conditionals. Can you move the call to the hook outside and just use the value here?